src/moaicore/MOAIDraw.h

```diff
@@ -68,6 +69,7 @@ class MOAIDraw :
 	//----------------------------------------------------------------//
 	static void			DrawLuaParams			( lua_State* L, u32 primType );
 	static void			DrawLuaArray			( lua_State* L, u32 primType );
+	static void			DrawFromLua				( lua_State* L, u32 primType );
```

Comment:
SUPER nitpicky, so ignore if it's unnecessary, but should this be indented so far?